packages/data/src/components/use-select/test/index.js

```diff
@@ -942,7 +942,7 @@ describe( 'useSelect', () => {
 			expect( rendered.getByRole( 'status' ) ).toHaveTextContent( 1 );
 
 			// initial render + subscription check + rerender with isAsync=false
-			expect( selectSpy ).toHaveBeenCalledTimes( 3 );
+			expect( selectSpy ).toHaveBeenCalledTimes( 4 );
```

Comment:
This happens because the `useIsomorphicLayoutEffect` dependency `isAsync` changes, which seems normal. Comment needs updating.

Reply:
If the `isSyncing = isAsync != null` from https://github.com/WordPress/gutenberg/pull/32091/files#r986891973 is implemented, then this test need not change. There will be no new `mapSelect` call.